openvdb/cmd/CMakeLists.txt

```diff
@@ -0,0 +1,296 @@
+# LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+# DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+# THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# IN NO EVENT SHALL THE COPYRIGHT HOLDERS' AND CONTRIBUTORS' AGGREGATE
+# LIABILITY FOR ALL CLAIMS REGARDLESS OF THEIR BASIS EXCEED US$250.00.
+#
+#[=======================================================================[
+
+  CMake Configuration for OpenVDB Binaries
+
+#]=======================================================================]
+
+PROJECT ( OpenVDBBinaries )
+CMAKE_MINIMUM_REQUIRED ( VERSION 3.3 )
+# Monitoring <PackageName>_ROOT variables
+IF ( POLICY CMP0074 )
+  CMAKE_POLICY ( SET CMP0074 NEW )
+ENDIF ()
+
+###### OpenVDB Binary Component Options
+
+OPTION ( OPENVDB_BUILD_VDB_LOD "Build vdb_lod" ON )
+OPTION ( OPENVDB_BUILD_VDB_PRINT "Build vdb_print" ON )
+OPTION ( OPENVDB_BUILD_VDB_RENDER "Build vdb_render" ON )
+OPTION ( OPENVDB_BUILD_VDB_VIEW "Build vdb_view" ON )
+
+#########################################################################
+
+MESSAGE ( STATUS "----------------------------------------------------" )
+MESSAGE ( STATUS "----------- Configuring OpenVDBBinaries ------------" )
+MESSAGE ( STATUS "----------------------------------------------------" )
+
+##########################################################################
+
+# Collect and configure lib dependencies
+
+IF ( NOT OPENVDB_BUILD_CORE )
+  FIND_PACKAGE ( OpenVDB REQUIRED )
+  SET ( OPENVDB_LIB OpenVDB::openvdb )
+ELSE ()
+  SET ( OPENVDB_LIB openvdb_shared )
+ENDIF ()
+
+FIND_PACKAGE ( IlmBase REQUIRED COMPONENTS Half Iex IlmThread )
+FIND_PACKAGE ( TBB REQUIRED COMPONENTS tbb )
+FIND_PACKAGE ( ZLIB REQUIRED )
+
+IF ( USE_LOG4CPLUS )
+  FIND_PACKAGE ( Log4cplus REQUIRED )
+ENDIF ()
+
+IF ( USE_BLOSC )
+  FIND_PACKAGE ( Blosc ${MINIMUM_BLOSC_VERSION} REQUIRED )
+ENDIF ()
+
+FIND_PACKAGE ( Boost ${MINIMUM_BOOST_VERSION} REQUIRED COMPONENTS iostreams system )
+
+IF ( UNIX )
+  FIND_PACKAGE ( Threads REQUIRED )
+ENDIF ()
+
+# Set core deps. Note that the order here is important. If we're building against
+# Houdini 17.5 we must include OpenEXR and IlmBase deps first to ensure the normal
+# namespaced headers are used over the Houdini 17.5 ones (which will be imported
+# through targets like tbb which are still deployed with Houdini)
+
+SET ( OPENVDB_BINARIES_DEPENDANT_LIBS
+  ${OPENVDB_LIB}
+  Boost::iostreams
+  Boost::system
+  IlmBase::Half
+  )
+
+IF ( USE_LOG4CPLUS )
+  LIST ( APPEND OPENVDB_BINARIES_DEPENDANT_LIBS Log4cplus::log4cplus )
+ENDIF ()
+
+IF ( USE_BLOSC )
+  LIST ( APPEND OPENVDB_BINARIES_DEPENDANT_LIBS Blosc::blosc )
+ENDIF ()
+
+LIST ( APPEND OPENVDB_BINARIES_DEPENDANT_LIBS
+  TBB::tbb
+  ZLIB::ZLIB
+  Threads::Threads
+  m
+  stdc++
```

Comment:
Haven't tested, but just flagging that this might be an issue on later versions of OSX, because that uses libc++ instead of libstdc++ by default...

Reply:
I've tested on 10.14.3 Mojave (see test envs in pr description)

Reply:
I've actually ended up remove these, they don't seem to be required anymore probably due to setting the CXX standard